index.css

```diff
@@ -153,21 +157,48 @@ footer h2 {
   margin-top: 90px;
 }
 
+.box1 {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
 .box {
-  background-image: url(../Pictures/profesionalart.png);
+  background-image: url(./Pictures/profesionalart.png);
+  background-repeat: no-repeat;
+}
+
+.box2 {
+  background-image: url(./Pictures/profesionalart.png);
+  background-repeat: no-repeat;
+}
+
+.box3 {
+  background-image: url(./Pictures/profesionalart.png);
+  background-repeat: no-repeat;
+}
+
+.box4 {
+  background-image: url(./Pictures/profesionalart.png);
```

Comment:
- [ ] [OPTIONAL]In the Mobile version the image is not showing, there is a typo error. Kindly change `Pictures` to `pictures`.

Reply:
I miss that point going to make the changes requested thank you so much for your CR

Reply:
@Deepakdanger coach what browser CR use or how i can see if my code is running well in my windows work